.github/workflows/ci.yml

```diff
@@ -221,8 +221,15 @@ jobs:
         with:
           profile: minimal
           toolchain: stable
-      - name: cargo test --locked --all-targets --workspace --exclude fuel-indexer-tests
-        run: cargo test --locked --all-targets --workspace --exclude fuel-indexer-tests
+          target: wasm32-unknown-unknown
+
+      - uses: Swatinem/rust-cache@v1
```

Comment:
- Can we break your CI additions out into their own `job`? (idk something like `forc-index-test`)
- This would have the same `if` as the current `check-forc-index` job

```yaml
if: needs.check-is-semver-branch.outputs.is_semver_branch != 'true'
```

So that we don't run it when cutting a new release or creating a new release PR